.github/workflows/release.yml

```
@@ -67,10 +37,26 @@ jobs:
       - name: Outdated files, run `pnpm build` and commit them
         uses: ./.github/actions/require-empty-diff
 
+      - name: Check for pre.json file existence
+        id: check_files
+        uses: andstor/file-existence-action@v2.0.0
+        with:
+          files: ".changeset/pre.json"
+
       - name: Set deployment token
         run: npm config set '//registry.npmjs.org/:_authToken' "${NPM_TOKEN}"
+
+      # https://github.com/changesets/action#with-publishing
+      - name: Handle Release Pull Request or Publish to npm
+        id: changesets
+        if: steps.check_files.outputs.files_exists == 'false'
+        uses: changesets/action@v1
+        with:
+          title: "chore: Version packages 🔖"
+          commit: "chore: Version packages 🔖"
```

```suggestion
          title: "chore: version packages 🔖"
          commit: "chore: version packages 🔖"
```

ocd requires me to lowercase this